src/frontend/src/optimizer/plan_node/logical_expand.rs

```diff
@@ -94,14 +94,30 @@ impl PlanTreeNodeUnary for LogicalExpand {
                     .collect_vec()
             })
             .collect_vec();
-        let (mut mapping, new_input_col_num) = input_col_change.into_parts();
```

Comment:
IIUC `new_input_col_num` is the offset we originally used.

After this PR we use `new_in_len`. What's the difference?

Reply:
On their naming, the meaning of `new_input_col_num` is exactly the same as that of `new_in_len`, the difference is in what we assign to them.